dvc/command/ls/ls_colors.py

```diff
@@ -0,0 +1,28 @@
+import dircolors
+
+from dvc.compat import fspath
+
+
+class LsColors(dircolors.Dircolors):
+    def __init__(self, load=True):
+        super().__init__(load)
+
+        if "dvc" not in self._codes:
+            self._codes["dvc"] = "01;31"  # Red
+
+        if ".dvc" not in self._extensions:
+            self._extensions[".dvc"] = "01;33"  # Yellow
```

Comment:
These might overlap with the theme and so they will lose their value. Maybe let's color them as regular files/dirs? For marking dvc outputs we might figure some other way later.

Reply:
I've disabled coloring `.dvc` files and dvc-outputs by default